From 70068c2cd51d98a90bf6ea54e454328133db8f10 Mon Sep 17 00:00:00 2001
From: Steven Rostedt <srostedt@redhat.com>
Date: Fri, 22 Feb 2013 12:07:16 -0500
Subject: [PATCH 288/288] Linux 3.4.39-rt53 REBASE

---
 localversion-rt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 700c857..e6c0dc6 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt8
+-rt53
-- 
1.7.10.4

